chapters/arrays.tex

```diff
@@ -1417,12 +1417,12 @@ \section{Empty Arrays}\label{empty-arrays}
 
 Empty matrices can be constructed with the fill function.  For example:
 \begin{lstlisting}[language=modelica]
-Real A[:,:] = fill(0.0, 0, 1); // a Real 0 x 1 matrix
-Boolean B[:, :, :] = fill(false, 0, 1, 0); // a Boolean 0 x 1 x 0 matrix
+Real A[:,:] = fill(0.0, 0, 1); // an empty Real 0 x 1 matrix
+Boolean B[:, :, :] = fill(false, 0, 1, 0); // an empty Boolean 0 x 1 x 0 matrix
```

Comment:
Did you verify that the source code line doesn't wrap?

Reply:
No, I did not.